profiles/consent-state/terraform/consent/elasticsearch-init.tf

```diff
@@ -0,0 +1,10 @@
+resource "null_resource" "elasticsearch_init" {
+  triggers = {
+    always = "${uuid()}"
+  }
+  provisioner "local-exec" {
+    command = <<EOF
+curl -X PUT "http://${var.elasticsearch_hostname}:9200/ontology/" -H 'Content-Type: application/json' -d '{"settings":{"index":{"number_of_replicas":1}}}'
```

Comment:
can you make `number_of_replicas` a variable?

Reply:
Oh yeah I was actually gonna do that and then forgot